src/pbs_installer/_versions.py

```diff
@@ -2,7 +2,7 @@
 from __future__ import annotations
 from ._utils import PythonVersion
 
-PYTHON_VERSIONS: dict[PythonVersion, dict[tuple[str, str], tuple[str, str | None]]] = {
```

Comment:
I know the file says "DO NOT EDIT", but I didn't want to have some kind of additional effects when updating the file. I can take this change back. I have also fixed the template, so it should be fine after the next file regeneration.